src/pages/Chat/state.tsx

```diff
@@ -120,6 +133,55 @@ export const ChatContainer: React.SFC<
                 },
               })
             },
+            setHasCurrentInsurance: (
+              event: React.ChangeEvent<HTMLSelectElement>,
+            ) => ({ state, setState }: EffectProps<State>) => {
+              const newState: Partial<State> = {
+                currentInsurance: cond([
```

Comment:
I'm having trouble reading this 😕

Reply:
):

Reply:
ill refactor

Reply:
i have a PR to remove it completely coming up after these are merged. I see no reason to pull it in if we only use 4 functions from it.